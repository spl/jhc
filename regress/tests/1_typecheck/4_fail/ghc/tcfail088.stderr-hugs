ERROR "tcfail088.hs":9 - Syntax error in type expression (unexpected keyword "forall")
