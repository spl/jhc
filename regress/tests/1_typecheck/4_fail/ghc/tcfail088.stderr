
tcfail088.hs:9:19:
    Illegal polymorphic or qualified type: forall s. T s a
    In the instance declaration for `Ord (forall s. T s a)'
